### PR TITLE
Add custom SDF field filter to ob_filter tool 

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

**Comment by bgruening** [comment]
there is a merge conflict here

File: chemicaltoolbox/openbabel/ob_filter.xml [hunk 1]
```diff
@@ -117,6 +122,11 @@
               <param name="atoms_min" type="integer" value="" label="Minimum number of atoms"/>
               <param name="atoms_max" type="integer" value="" label="Maximum number of atoms"/>
             </when>
+            <when value="field">
+              <param name="field_name" type="text" value="" label="Name of the SDF property to filter by"/>
```
Comment (bgruening) [inline chemicaltoolbox/openbabel/ob_filter.xml:126]: please add here a sanatizer ... only letters and numbers?

**Comment by bgruening** [comment]
Added new version.

**Comment by simonbray** [comment]
Thanks @bgruening

**Pull request merged**